scrubble-client/src/html/game.html

```diff
@@ -1,5 +1,61 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Scrubble</title>
+  </head>
   <body>
-    <div id="pixi-application"></div>
+    <header class="header">
+      <p class="header__logo">Scrubble</p>
+    </header>
+    <section class="room-code">
+      <h2 class="room-code__label">Room Code:</h2>
+      <h2 class="room-code__code">WXYZ</h2>
+    </section>
+    <section class="players">
+      <div class="players__current-turn" id="players__1">
+        <h3 class="players__name" id="players__1-name">Player 1:</h3>
+        <h3 class="players__score" id="players__1-score">24</h3>
+      </div>
+      <div class="players__player" id="players__2">
+        <h3 class="players__name" id="players__2-name">Player 2:</h3>
+        <h3 class="players__score" id="players__2-score">20</h3>
+      </div>
+      <div class="players__player" id="players__3">
+        <h3 class="players__name" id="players__3-name">Player 3:</h3>
+        <h3 class="players__score" id="players__3-score">18</h3>
+      </div>
+      <div class="players__player" id="players__4">
+        <h3 class="players__name" id="players__4-name">Player 4:</h3>
+        <h3 class="players__score" id="players__4-score">28</h3>
+      </div>
+      <div class="players__player" id="players__5">
+        <h3 class="players__name" id="players__5-name">Player 5:</h3>
+        <h3 class="players__score" id="players__5-score">30</h3>
+      </div>
+      <div class="players__player" id="players__6">
+        <h3 class="players__name" id="players__6-name">Player 6:</h3>
+        <h3 class="players__score" id="players__6-score">12</h3>
+      </div>
+    </section>
+    <section class="game-board">
+      <h3 class="game-board__whose-turn">It's Mitch's Turn</h3>
+      <div class="game-board__board" id="pixi-application"></div>
+      <div class="game-board__your-tiles">
```

Comment:
Not necessarily super important for now, but worth mentioning that these tiles will probably need to be incorporated into the game canvas element in order to take full advantage of that sweet sweet drag-n-drop goodness

Reply:
yes agreement